static/style.css

```diff
@@ -100,15 +98,72 @@ footer a:hover {
   display: flex;
 }
 
-.touchpad p {
+.touchpad {
+  background-color: silver;
+  border: 1px solid lightgray;
+  float: left;
+  width: 82%;
+  color: #535050;
+  border-top-left-radius: 10px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+/* .touchpad p {
```

Comment:
Clean up the comments?

Reply:
Ah, also this file is generated with Sass! You will need to make changes there and compile it into css for now 
https://github.com/shubham1172/remote-play/blob/main/CONTRIBUTING.md#converting-scss-to-css

I will clean this up later, but please bear with this for now!